app/javascript/components/Profile.js

```diff
@@ -224,7 +224,7 @@ export default class Profile extends Component {
             <div className="flex-col w-1/2 mt-10 p-5 bg-gray m-auto bg-grey-lightest border border-grey rounded">
               <UpdateAccessToken hasAccessToken={data.me.hasAccessToken} />
               <UpdateGithubUsername githubUsername={data.me.githubUsername} />
-              <UpdateGithubOrg orgs={MOCK_ORGS} />
```

Comment:
Can you pls remove the [MOCK_ORGS const](https://github.com/oddballio/magnifier/blob/master/app/javascript/components/Profile.js#L10-L21), también